scripts/lib/CIME/case/check_input_data.py

```diff
@@ -129,7 +166,8 @@ def stage_refcase(self, input_data_root=None, data_list_dir=None):
             os.remove(os.path.join("Buildconf","refcase.input_data_list"))
     return True
 
-def check_input_data(case, protocol="svn", address=None, input_data_root=None, data_list_dir="Buildconf", download=False):
+def check_input_data(case, protocol="svn", address=None, input_data_root=None, data_list_dir="Buildconf",
+                     download=False, chksum=False, user=None, passwd=None):
```

Comment:
Please add documentation for the new chksum argument. (Ideally there would be documentation for all arguments.)